packages/website-frontend/src/app/repository/repository.service.spec.ts

```diff
@@ -22,35 +45,45 @@ describe('RepositoryService', () => {
 
   describe('getRepositories()', () => {
 
-    it('should return an Observable<Repository[]>', () => {
-      const mockedRepositories = [
-        {
-          slug: 'stryker-mutator/stryker-badge',
-          origin: 'https://www.github.com',
-          owner: 'stryker-mutator',
-          name: 'stryker-badge',
-          enabled: true
-        }, {
-          slug: 'stryker-mutator/stryker',
-          origin: 'https://www.github.com',
-          owner: 'stryker-mutator',
-          name: 'stryker',
-          enabled: true
-        }, {
-          slug: 'stryker-mutator/stryker-jest-runner',
-          origin: 'https://www.github.com',
-          owner: 'stryker-mutator',
-          name: 'stryker-jest-runner',
-          enabled: false
-        }
-      ];
-      
-      repositoryService.getRepositories().subscribe((repositories) => {
-        expect(repositories.length).toBeGreaterThan(0);
+    it('should send a GET request to /api/user/repositories and return an Observable<Repository[]>', () => {
+      repositoryService.getRepositories().subscribe((repositories: Repository[]) => {
+        expect(repositories.length).toBe(3);
       });
+      let request = httpMock.expectOne('api/user/repositories');
+      expect(request.request.method).toBe('GET');
+      request.flush(mockedRepositories);
+      httpMock.verify();
+    });
+
+  });
+
+  describe('enableRepositories()', () => {
+
+    it('should send a PATCH request with enabled = false', () => {
+      repositoryService.enableRepository(mockedRepositories[0].slug, false)
+        .subscribe((response: EnableRepositoryResponse) => {
+          // TODO: change this assert
+          expect(response).toBeTruthy();
```

Comment:
I was working on a working version first, and I couldn't seem to get rid of this issue. The issue only seems to exist in this specific test, and should not pose any real problem in production. That's I skipped it for now, and continued with other work. The TODO is there as a reminder, because I think we should still test it properly.